components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -81,7 +81,7 @@ Comment getComment(ApiTypeWrapper apiTypeWrapper, String commentId, Integer repl
      * @return Comments
      * @throws APIManagementException if failed to get comments for identifier
      */
-    boolean editComment(ApiTypeWrapper apiTypeWrapper, String commentId, Comment comment) throws APIManagementException;
+    boolean editComment(String apiTypeWrapper, String commentId, Comment comment) throws APIManagementException;
```

Comment:
samehere

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/10614/commits/b561990adfc350c2aec76fc8e72c88c6ad896f15